File: backend/src/main/java/com/hrift/backend/models/ApplicationModel.java
package com.hrift.backend.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import org.springframework.context.annotation.Bean;

@Data
public class ApplicationModel {

   private Long offerId;
//    private String skills;
//    private String cvFileName;
    private Long exp;

    private String FirstName;
    private String LastName;
    private String phone;
    private String disponibility;

}

File: frontend/src/app/models/applicationmodel.ts
import { Offersmodel } from "./offersmodel";

export class ApplicationModel {
    id: number = 0;
    firstName: string = '';
    lastName: string = '';
    phone: string = '';
    disponibility: string = '';
    offerId: number = 0;
    offer: Offersmodel = new Offersmodel()
    score: number = 0;
    exp: number= 0;
    createdAt: string='';
    skills: string[] = [];
}